cypress/e2e/learnstore/manageAccessToLearnstore.feature

```diff
@@ -0,0 +1,41 @@
+@release
+Feature: Learnstore - Activating and deactivating access for students
```

Comment:
Same stuff either its `Learnstore`, `learnStore`, `Learning_Store` whatever. make it consistent in all places incl. naming of files. Don't want to mention in every places as it makes redundant. So only wrote here

Reply:
I understand that point.
There were different names before, even som Denglish stuff, something like LerningStore .. so I changed some but not all.
I'll try to change more.